tests/test_interaction.py

```diff
@@ -1,7 +1,5 @@
 import unittest
 from simulation import person
-from simulation import container
-import plotly.graph_objects as go
 
 
 class InfectedAndSusceptibleInteractionTestCase(unittest.TestCase):
```

Comment:
Where are tests for each big method from `app.py`? I am not talking about plots etc. but rather about functionality, `R0` `q` etc.